lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -61,6 +61,24 @@ mapped_vlan:
     config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
     default_value: ''
 
+mt_full_support:
+  # This is only used for determining support for Multi-Tenancy Full
+  kind: boolean
+  /N7/:
+    default_only: true
+  else:
+    # this feature is always off on these platforms and cannot be changed
+    default_only: false
+
+mt_lite_support:
```

Comment:
Just checked with my team, 5k/6k are mt-lite, so you may wanna add them under mt-lite here.